test/e2e/helpers.js

```diff
@@ -60,15 +62,16 @@ async function withFixtures(options, testSuite) {
   const {
     dapp,
     fixtures,
+    localNode = 'ganache',
     ganacheOptions,
+    anvilOptions,
```

Comment:
[TODO] once [this PR ](https://github.com/MetaMask/metamask-extension/pull/30058/)is merged, we'll be able to consolidate this into 1 variable called `localNodeOptions`

Reply:
:heavy_check_mark:  done